sql/v2/runtime/test/tck/user_defined_functions.yaml

```diff
@@ -0,0 +1,27 @@
+name: User defined functions
+tests:
+  - name: HASPREFIX (1)
+    expression: "HASPREFIX('abcdef', 'ab')"
+    result: true
+  - name: HASPREFIX (2)
+    expression: "HASPREFIX('abcdef', 'abcdef')"
+    result: true  
+  - name: HASPREFIX (3)
+    expression: "HASPREFIX('abcdef', '')"
+    result: true  
+  - name: HASPREFIX (4)
+    expression: "HASPREFIX('abcdef', 'gh')"
+    result: false  
+  - name: HASPREFIX (5)
+    expression: "HASPREFIX('abcdef', 'abcdefg')"
+    result: false
+
```

Comment:
remove newline?

Reply:
The existing sql tck yaml files use new line to separate tests cases. 
https://github.com/cloudevents/sdk-go/blob/main/sql/v2/test/tck/string_builtin_functions.yaml#L12-L18

I was following their format. I think the new line between consecutive tests cases should be kept but the new line at the end of the file should be removed.